docs/concepts/vega-chain/network.md

```diff
@@ -1,27 +1,128 @@
+
+It's possible to submit a large volume of transactions at one time, as long as the proof-of-work is calculated by the wallet adequately.
 
-There are several reasons for this decision, including: 
+A transaction with a missing or incorrect proof is rejected, as is any transaction that uses an already-used PoW calculation. A public key that sends transactions with faulty proof-of-work calculations is banned for 30 seconds or 1/48th of an epoch, whichever is greater.
 
-- Limited network life makes it efficient to upgrade the protocol by starting again, as it avoids the need to deal with multiple versions of the code. Upgrades to a running chain need to respect and be able to recalculate the deterministic state for earlier blocks, so all versions of critical code must remain in the system. 
-- It allows for rapid iteration, as the ability to start new chains for new features is simpler.
-- Once there is a network with trading, the thousands of transactions per second will generate a lot of data. Given that most instruments expire, this allows for new markets to be created on a new chain, allowing an old market/chain to come to an end rather than keeping the history and data forever.
+See a full list of the network parameters used for PoW in the [spam protection parameters table](#spam-protection-parameters).
 
-### Network checkpoints 
+:::note Go deeper
+[Spam protection PoW ↗](https://github.com/vegaprotocol/specs/blob/master/protocol/0072-SPPW-spam-protection-PoW.md): Read the spec for implementation details and how this mechanism works in particular circumstances.
+:::
+
+### Spam policy enforcement
+Messages that don't follow the anti-spam rules are rejected, either pre- or post-block.
+
+Pre-block rejection: A transaction is rejected before it enters the validators' mempool. The case for a rejection can only be determined based on transactions that were in the previous block, and not on any other transactions received by a validator but not yet put into a block. Once a block is scheduled, all transactions in the mempool are verified. If a transaction fails the applicable spam rules set by the [parameters](#spam-protection-parameters) below, it's not processed and is removed.
+
+Post-block rejection: A transaction makes it into the block, but is rejected before it's passed to the application layer. This allows for more fine-grained policies because more information is available, but the offending transaction will take up space in the block. If a transaction fails the spam rules set by the [parameters](#spam-protection-parameters) below, it is rejected and may trigger a spam proof of work threshold increase.
```

Comment:
>it is rejected and may trigger a spam proof of work threshold increase

the first part is correct, the second is not. It may cause the party to be banned and it may increase the min token requirement for submitting transactions but it may not do anything to pow threshold.

Reply:
@ze97286 Min token requirement is just increased for the individual pubkey, right?